x-pack/legacy/plugins/apm/public/components/shared/MetadataTable/SpanMetadata/index.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { SPAN_METADATA_SECTIONS } from './sections';
+import { Span } from '../../../../../typings/es_schemas/ui/Span';
+import { MetadataTable } from '..';
+
+interface Props {
+  span: Span;
+}
+
+export function SpanMetadata({ span }: Props) {
+  const clonedSpan: Span = JSON.parse(JSON.stringify(span));
```

Comment:
Would it be better to use lodash's `cloneDeep` here? I don't like using parse/stringify. Practically it's probably the same performance-wise but I'd prefer something more explicit.